### PR TITLE
Support for UltraSSD DataDisk

### PR DESCRIPTION
**Pull request by xpillons**

### CODE REVIEW

**Comment by xpillons** [comment]
@hmeiland can you please unit test that ?

File: pyazhpc/arm.py [hunk 1]
```diff
@@ -651,7 +654,10 @@ def _add_vm(self, cfg, r, vnet_in_deployment):
                 vmres["properties"]["proximityPlacementGroup"] = {
                     "id": f"[resourceId('Microsoft.Compute/proximityPlacementGroups','{rppgname}')]"
                 }
-            
+
+            if rstoragesku == "UltraSSD_LRS" :
+                vmres["properties"]["additionalCapabilities"] = { "ultraSSDEnabled": "true" }
```
Comment (edwardsp) [inline pyazhpc/arm.py:659]: is this supposed to be a string or bool value?

Comment (edwardsp) [reply pyazhpc/arm.py:659]: @hmeiland 
fyi here is the rest api - https://docs.microsoft.com/en-us/rest/api/compute/virtualmachines/createorupdate#additionalcapabilities - which looks to be of type bool

Comment (hmeiland) [reply pyazhpc/arm.py:659]: yep: agree

**Comment by hmeiland** [comment]
> you need to set as **True** (with a capital T) rather than **true** as this is in Python

yes, updated with newer commit

**Comment by xpillons** [comment]
@edwardsp @hmeiland this is all tested now with the latest fix about zones and public IP. pipeline passed so it can be approved

**Pull request merged**